server/src/main/proto/dict_pb.proto

```diff
@@ -0,0 +1,36 @@
+syntax="proto2";
+
+package dst.server.protocol.pb;
+
+option java_package="org.dst.server.generated";
+option java_outer_classname="DictProtocol";
+option cc_generic_services = true;
+
+//Dict
+message DstDict {
+    repeated string keys = 1;
+    repeated string dict = 2;
+}
+
+message DictPutRequest {
+    required string key = 1;
+    optional DstDict dict = 2;
```

Comment:
This shouldn't be optional.

When you put a dict, value is not allowed to be set to NIL.